helm/charts/hpe-csi-driver/values.yaml

```diff
@@ -4,8 +4,8 @@
 
 # Control CSP Service and Deployments for HPE storage products
 disable:
-  nimble: false
-  primera: false
+  hpe-alletra-6000-and-nimble: false
+  hpe-alletra-9000-primera-and-3par: false
```

Comment:
"hpe" is already qualified. But would desire the following user facing interface:
```
disable:
  nimble: false
  primera: false
  alletra6000: false
  alletra9000: false
```

Reply:
Will update